common/compositor/va/varenderer.cpp

```diff
@@ -305,10 +315,17 @@ bool VARenderer::UpdateCaps() {
     VAProcFilterParameterBufferColorBalance cbparam;
     VAProcFilterParameterBuffer sharpparam;
 
-    for (ColorBalanceCapMapItr itr = colorbalance_caps_.begin();
-         itr != colorbalance_caps_.end(); itr++) {
+    for (auto itr = colorbalance_caps_.begin(); itr != colorbalance_caps_.end();
+         itr++) {
+      bool use_default =
+          itr->second.use_default_ &&
+          itr->second.value_ != itr->second.caps_.range.default_value;
```

Comment:
shouldn't this be "=="

Reply:
no. after restoring the default value, filter is not required any more.